tests/unit/components/labelPrinting/LabelPrintingForm.spec.js

```diff
@@ -0,0 +1,199 @@
+import LabelPrintingForm from '@/components/labelPrinting/LabelPrintingForm'
```

Comment:
It feels like things are being tested twice here. You have a thorough e2e test which fills in all of the fields and then they are being checked again here.

Reply:
Are you just talking about the `wrapper.find('#...').exists()` tests here? I think I'd agree about those. But generally speaking I don't see harm in covering stuff via both unit tests and e2e as they are great at surfacing different things.

Reply:
cool I will remove the `wrapper.find('#...').exists()` but leave the rest